src/main/java/ConsoleMessage.java

```diff
@@ -0,0 +1,12 @@
+public class ConsoleMessage extends Message {
+
+    String className = this.getClass().getSimpleName();
+
+    @Override
+    public String getMessage() {
+
+        return "This is class " + className;
```

Comment:
tu można bezpośrednio wywołać `this.getClass().getSimpleName()` ale generalnie jest ok